fr.nytuo.cosmiccomics.yml

```diff
@@ -0,0 +1,57 @@
+app-id: fr.nytuo.cosmiccomics
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=host
+  - --device=dri
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: cosmiccomics
+    buildsystem: simple
+    build-options:
+      env:
+        XDG_CACHE_HOME: /run/build/cosmiccomics/flatpak-node/cache
+        npm_config_cache: /run/build/cosmiccomics/flatpak-node/npm-cache
+        npm_config_offline: "true"
+    build-commands:
+      # Install npm dependencies
+      - npm install --offline
+      - chmod a+x ./node_modules/7zip-bin/linux/x64/7za
```

Comment:
this will work great on aarch64.

Reply:
Thanks, didnt notice it, I've reworked the strategy to avoid chmod and platform specific command